examples/sample_01/main.tf

```diff
@@ -0,0 +1,92 @@
+provider "google" {
+  credentials = file(var.credentials_file)
+  project = var.project
```

Comment:
it is better if you specify these parameters on the resources explicitly than here. 
This can lead to confusing errors, if you work in an environment with multiple projects and locations